packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2979,6 +2985,15 @@ class BuildOwner {
   }
 }
 
+/// Concrete implementation of [BuildOwner] for the widgets library.
+class WidgetsBuildOwner extends BuildOwner {
```

Comment:
maybe DefaultBuildOwner?

Reply:
or maybe RawBuildOwner/BuildOwner?

Reply:
or maybe don't even have WidgetsBuildOwner -- just change the call site to create the FocusManager.